tools/sv-report

```diff
@@ -352,22 +359,21 @@ try:
         # find the number of tests that passed
         rts = data[r]["tests"]
         data[r]["total"]["tests"] = sum(
-                1 for t in rts if rts[t]["status"] in "test-passed")
+            1 for t in rts if rts[t]["status"] in "test-passed")
 
         # find the number of tags for which all the tests passed
         rtt = data[r]["tags"]
         data[r]["total"]["tags"] = sum(
-                1 for t in rtt if rtt[t]["status"] in "test-passed")
+            1 for t in rtt if rtt[t]["status"] in "test-passed")
 
     with open(args.template, "r") as f:
-        report = jinja2.Template(f.read(),
-                                 trim_blocks=True,
-                                 lstrip_blocks=True)
+        report = jinja2.Template(
+            f.read(), trim_blocks=True, lstrip_blocks=True)
 
     with open(args.out, 'w') as f:
-        f.write(report.render(report=data,
-                              database=database,
-                              revision=args.revision))
+        f.write(
+            report.render(
+                report=data, database=database, revision=args.revision))
```

Comment:
I am not a fan of the formatting changes made by `yapf`, but if the same settings are used by symbiflow in other repos, then I guess it is ok.